typed-racket-more/typed/images/icons.rkt

```diff
@@ -0,0 +1,212 @@
+#lang typed/racket/gui
+
+(require "logos.rkt" (submod "logos.rkt" digitama))
+(require typed/racket/draw)
+
+(require/typed
+ images/private/deep-flomap
+ [#:opaque Flomap-Material deep-flomap-material-value?])
```

Comment:
Sure, but either you should use the typed version in `images/private/deep-flomap` or the untyped version in `deep-flomap-untyped-parameters`. Using `require/typed` here just adds an extra set of contracts.